docs/README.md

```diff
@@ -8,7 +8,8 @@
   - [About the `schema` and `uiSchema` objects](building-a-form/about-the-schema-and-uischema-objects.md)
   - [Quick Start: Example `form.js` file](building-a-form/quick-start-example-formjs-file.md)
   - [Creating a form config file](building-a-form/creating-a-form-config-file.md)
-  - [Adding widgets and definitions](building-a-form/adding-widgets-and-definitions.md)
   - [Available form features and usage guidelines](building-a-form/available-form-features-and-usage-guidelines.md)
+  - [Common widgets](building-a-form/common-widgets.md)
```

Comment:
Are these titles useful? We now have three topics called "Common..." and it's concerning me. 😕

Reply:
I suppose this particular one should be called Available Widgets but the others seem good as Common